src/components/Ui/Dropdown.vue

```diff
@@ -100,4 +104,18 @@ li:hover {
   opacity: 1;
   transition-delay: 0.3s;
 }
+.search-dropdown {
+  top: -2.5px;
```

Comment:
We should avoid custom class, we can use Primer classes instead here.

Reply:
@bonustrack Which attribute did you mean here?

Reply:
Can you `mt-n1 text-left border-right` for example